src/components/MagicImage/MagicImage.js

```diff
@@ -0,0 +1,61 @@
+import {useRef} from 'react';
+import styled from 'styled-components';
+import {keyframes, css} from 'styled-components';
+
+import foobar from '../../images/sprites/foobarSprite1.png';
+import {FrontFoobarImg} from '../../stylesheet/StyledImages.js';
+
+export default function MagicImage({currentBackgroundImage, isRoomEnlighten}) {
+  const fader = useRef();
+
+  return (
+    <>
+      <FramingDiv>
+        <FaderDiv ref={fader} fadeSelect={isRoomEnlighten ? fadeOutSelect : noFade}></FaderDiv>
+        {currentBackgroundImage}
+        <FrontFoobarImg src={foobar} alt="" width="172" height="438" />
+      </FramingDiv>
+    </>
```

Comment:
You don't need the fragments here:
```suggestion
      <FramingDiv>
        <FaderDiv ref={fader} fadeSelect={isRoomEnlighten ? fadeOutSelect : noFade}></FaderDiv>
        {currentBackgroundImage}
        <FrontFoobarImg src={foobar} alt="" width="172" height="438" />
      </FramingDiv>
```